### ISSUE TITLE
About Create Snapshot Into Hive

### ISSUE THREAD

**Issue by BookXHD** [metatron-app/metatron-discovery#2057]
hello,
I've configed hdfs and hive connection, and when i create a snapshot into Hive, it works well, but it appears error when i check the table in Hive:
![image](https://user-images.githubusercontent.com/33343442/57667700-23f44280-7637-11e9-8ba1-80b409b2f602.png)
And when i check the snapshot created in the snapshot list of metatron:
[console_error.txt](https://github.com/metatron-app/metatron-discovery/files/3175652/console_error.txt)
If it was a BUG, i think the bug source should be the process of creating snapshot into Hive.

**Comment by kyungtaak** [comment]
@BookXHD Can you tell me which version of hadoop and hive you are using?

**Comment by BookXHD** [comment]
@kyungtaak ok, hadoop v3.0.0 and hive v2.1.1, and both deployed in CDH 6.1.1

**Comment by BookXHD** [comment]
@kyungtaak And here is my test data:
[small4.xlsx](https://github.com/metatron-app/metatron-discovery/files/3176479/small4.xlsx)

**Comment by kyungtaak** [comment]
@BookXHD Thank you for your information. My colleague can confirm it. 

@joohokim1 Can you check this issue?

**Comment by joohokim1** [comment]
@kyungtaak @BookXHD Yes, I'm on it.

**Comment by joohokim1** [comment]
@BookXHD @kyungtaak In my environment, creating and viewing the snapshot from [small4.xlsx](https://github.com/metatron-app/metatron-discovery/files/3176479/small4.xlsx) works fine.
So, I'm going to build the same environment to produce the bug, it might take some time.

**Comment by BookXHD** [comment]
@joohokim1 thank you, and by the way, what is the your environment which could work fine ?

**Comment by joohokim1** [comment]
@BookXHD It's Apache Hadoop 2.7.3 + Apache Hive 2.3.0. They are from their own official sites. I ran on MacOS Mojave. May I ask the OS where you tested? (I guessed it's linux, so I'm going with a CentOS docker.)

**Comment by BookXHD** [comment]
@joohokim1  I tested on MacOS Mojave too.

**Comment by eldorado0402** [comment]
@BookXHD Could you tell me the senario more detail? You just upload the excel file and create snaphot?
Is there any apply rules?

**Comment by BookXHD** [comment]
@eldorado0402  yes, i just upload the small4.xlsx as a dataset, and without applying any rules, i create  a snaphot in hive.And it is all goot up to this step, but when i check the hive table or the snaphot, they are all not good.

**Comment by eldorado0402** [comment]
@BookXHD I tested the senario in my local pc including hadoop 3.0.0. But the problem was't happended. I will try to do it again in cdh 6.X and share the result. Thanks

**Comment by BookXHD** [comment]
@eldorado0402 i can location the detail of this error now, it occurs when using the OCR(Textfile is not working either, with all of messy code), i found that 2 things:
1.the error"ArrayIndexOutOfBoundsException"is a kown typical error with OCR, but aftering search a lot, i still can not solve it.
2.My hive could use MR or Spark, but not Tez, will it make influence?

**Comment by joohokim1** [comment]
@BookXHD You mean CSV-based Hive does work?

**Comment by joohokim1** [comment]
@BookXHD 
Sorry, we tried best. I hope we can see this issue in further development soon.
Closing.
